lib/plugins/slack/attachements.js

```diff
@@ -93,10 +93,20 @@ module.exports = {
       let color = 'danger';
 
       const limitMetric = metrics[options.limitMetric];
-      if (limitMetric) {
+      if (limitMetric && options.limitMetric === 'coach') {
         if (limitMetric.metric > options.limitError) {
           color = 'good';
-        } else if (limitMetric.metric > options.limitWarning) {
+        }
+        else if (limitMetric.metric > options.limitWarning) {
+          color = 'warning';
+        }
+      }
+      // SpeeddIndex/firstVisualChange
+      else if (limitMetric) {
+        if (limitMetric.metric < options.limitError) {
```

Comment:
yes but the default values in the cli is kind of a mess (error higher than warning for coach score), that's my comment about 5.0 or we just fix that now.